freqtrade/exchange/exchange.py

```diff
@@ -689,7 +689,16 @@ def fetch_dry_run_order(self, order_id) -> Dict[str, Any]:
     # Order handling
 
     def create_order(self, pair: str, ordertype: str, side: str, amount: float,
-                     rate: float, params: Dict = {}) -> Dict:
+                     rate: float, time_in_force: str = 'gtc') -> Dict:
+
+        if self._config['dry_run']:
+            dry_order = self.create_dry_run_order(pair, ordertype, side, amount, rate)
+            return dry_order
+
+        params = self._params.copy()
+        if time_in_force != 'gtc' and ordertype != 'market':
+            params.update({'timeInForce': time_in_force})
+
```

Comment:
I think this change is useful, we can exclude the rest of this PR, but I think this change should bee made. It doesn't really add any benefit to leverage/shorting though so I'm not going to do it. I'll close the PR, rollback these commits locally, and give this no more thought